Snippets/Snippets_5/Snippets_5.csproj

```diff
@@ -368,9 +369,9 @@
     <Content Include="SCPluginsConfig\heartbeatsIntervalConfig.xml" />
     <Content Include="Transports\Mqmq\MessageQueue.config.xml" />
     <Content Include="Transports\RabbitMQ_[2.0,3.0]\rabbitmqconnection.config.xml" />
-    <Content Include="Transports\SqlServer\MultiDb.config.xml">
```

Comment:
I added it before and never intended to make it '.xml' file ;) Not sure if that matters? @SimonCropp ?